packages/builder/src/components/design/PropertiesPanel/ScreenSettingsSection.svelte

```diff
@@ -7,13 +7,17 @@
   import RoleSelect from "./PropertyControls/RoleSelect.svelte"
   import { currentAsset, store } from "builderStore"
   import { FrontendTypes } from "constants"
+  import sanitizeUrl from "builderStore/store/screenTemplates/utils/sanitizeUrl"
 
   export let componentInstance
   export let bindings
 
   function setAssetProps(name, value, parser) {
     if (parser && typeof parser === "function") {
+      console.log("before", value)
       value = parser(value)
+      console.log("after", value)
+      console.log("deepGet", deepGet($currentAsset, name))
```

Comment:
do we need these logs in here?

Reply:
no, I forgot to remove them, I updated my PR.